.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
```

Comment:
Could having something like

    # This is the EditorConfig (http://editorconfig.org/) coding style file for SOCI.

here helpful?